### PR TITLE
pat-inlinevalidation: Fix bug where drop downs would not get validated

### PR DESCRIPTION
**Pull request by frapell**
This PR fixes gh-971

### CODE REVIEW

**Comment by frapell** [comment]
@jenkins-plone-org please run jobs

File: mockup/patterns/inlinevalidation/pattern.js [hunk 1]
```diff
@@ -126,6 +126,12 @@ define([
             fset = $input.closest('fieldset').attr('data-fieldset'),
             fname = $field.attr('data-fieldname');
 
+        // XXX: When cloning a form, values from 'select' elements are not kept
+        //      so we copy them from the original form here.
+        $.each( $("select", $form), function ( k, v ) {
+          $('[name="'+v.name+'"]', $cloned_form).val($(v).val());
```
Comment (ale-rt) [inline mockup/patterns/inlinevalidation/pattern.js:132]: Nice catch!

**Comment by frapell** [comment]
@ale-rt Thanks for the suggestion... while I don't think z3cform would generate different types of fields with the same name, there's nothing wrong with being more specific :)

I'll merge after all tests are green

**Comment by frapell** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**